.github/workflows/acquia_cms_ci.workflow.yml

```diff
@@ -93,13 +93,20 @@ jobs:
           #coverage: xdebug
       - name: Execute PHPStan
         run: |
+          # Remove Acquia CMS modules.
+          # @todo: Remove module from below which you want to make Drupal 11 compatible.
+          composer remove "drupal/acquia_cms_headless" --no-install --no-update
+          # Remove acquia_cms_headless module.
+          rm -fr modules/acquia_cms_headless
+          # Unset the repository path.
+          composer config repositories.acquia_cms_headless --unset
           rm composer.lock
           composer install
           ./vendor/bin/phpstan analyze
   integrated_tests:
     if: ${{ github.event_name == 'pull_request' }}
     name: ${{ matrix.acms_job }} (${{ matrix.orca-job }})
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Is there any issue with `ubuntu-latest`?

Reply:
Yes, see below 👇 
Currently, we can’t use Github Actions CI to run Integrated Test / Isolated CI jobs for Drupal 11 because Drupal 11 is compatible with SQLite 3.45 (and above) and currently Ubuntu 24.04 only supports SQLite 3.45 and ubuntu-latest resolves to ubuntu 22.04. See this: https://github.com/actions/runner-images?tab=readme-ov-file#available-images